hack/get-machine-run-tests.sh

```diff
@@ -739,11 +739,14 @@ fi
 if [ "${USE_MONITORING:-false}" = true ] ; then
     cat > $runfile <<EOF
 set -euxo pipefail
-cd $OS_O_A_DIR
+cd $OS_A_C_J_DIR
 
-ANSIBLE_LOG_PATH=/tmp/ansible-monitoring.log ansible-playbook -vvv \
-  -i inventory/hosts.localhost \
-  playbooks/openshift-monitoring/config.yml \
+ANSIBLE_LOG_PATH=/tmp/ansible-monitoring.log ansible-playbook -vvv --become \
+  --become-user root \
+  --connection local \
+  --inventory sjb/inventory/ \
+  -e openshift_deployment_type=origin  \
+  $OS_O_A_DIR/playbooks/openshift-monitoring/config.yml \
```

Comment:
For consistency, the playbook should be the last argument on the command line, after any `-e` arguments